src/lib/utils/sveltekit-search-params.ts

```diff
@@ -146,8 +146,10 @@ export function searchParameters<T extends object>(
             debouncedUpdateTimeout = setTimeout(() => {
                 const hash = $page.url.hash;
                 const query = new URLSearchParams($page.url.searchParams);
+                const changedParams: string[] = [];
                 const toBatch = (query: URLSearchParams) => {
                     for (const field of Object.keys(value)) {
+                        const initialQuery = query.toString();
                         if ((value as any)[field] == undefined) {
```

Comment:
I know this was existing, but should it be triple equals?

Reply:
Yeah good point, updated there and in two other spots.